lib/adhearsion/foundation/object.rb

```diff
@@ -14,7 +14,12 @@ def pb_logger
 module Celluloid
   class ActorProxy
     def logger
-      Actor.call @mailbox, :logger
+      if current_actor = Thread.current[:celluloid_actor]
+        current_actor.bare_object.send :logger
```

Comment:
I don't understand this change. Could you elaborate why we need to inspect the current `Thread` vs just send to the `Actor`'s mailbox?